dataland-e2etests/build.gradle.kts

```diff
@@ -0,0 +1,70 @@
+plugins {
+    kotlin("jvm")
+    kotlin("plugin.spring")
+    id("org.openapi.generator") version "5.3.0"
+}
+
+java.sourceCompatibility = JavaVersion.VERSION_17
+
+val backendOpenApiSpecConfig by configurations.creating {
+    isCanBeConsumed = false
+    isCanBeResolved = true
+}
+
+dependencies {
+    implementation("org.junit.jupiter:junit-jupiter:5.8.2")
+    implementation("com.squareup.moshi:moshi-kotlin:1.13.0")
+    implementation("com.squareup.moshi:moshi-adapters:1.13.0")
+    implementation("com.squareup.okhttp3:okhttp:4.9.3")
+    backendOpenApiSpecConfig(project(mapOf("path" to ":dataland-backend", "configuration" to "openApiSpec")))
+}
+
+tasks.withType<org.jetbrains.kotlin.gradle.tasks.KotlinCompile> {
+    dependsOn("generateBackendClient")

```

Comment:
Ich denke wir sollten ins Wiki oder in eine readme.md schreiben, dass man `./gradlew :dataland-e2etests:generateBackendClient` ausführen muss, damit die IDE alle Klassen hat.